packages/terra-application-layout/package.json

```diff
@@ -22,18 +22,19 @@
   },
   "homepage": "https://github.com/cerner/terra-framework#readme",
   "devDependencies": {
-    "terra-app-delegate": "^2.7.0"
+    "terra-app-delegate": "^2.7.0",
+    "react-intl": "^2.4.0",
```

Comment:
We can drop the `react-intl` and `terra-base` devDependencies. Terra-base should already be provided by terra-base and then we want terra-base to manager the react-intl version.

Reply:
Sure can! https://github.com/cerner/terra-core/issues/1944